src/App.scss

```diff
@@ -403,8 +403,24 @@ body {
       #gait-suggestion-div {
          grid-row: 2; //timeline sub-grid
          grid-column: 2; //timeline sub-grid
-         //TODO <-----------------------------------------------------------------------------------
-
+         position: relative;
+         width: calc(100% - 12px);  //minus timeline track border and width of timeline thumb
```

Comment:
heads up, the timeline track border is 3px, so this should be - 6px (two times it's thickness)

Reply:
Fixed.